gradle.properties

```
@@ -18,9 +18,10 @@
 
 org.gradle.jvmargs=-Xmx6g -XX:+UseParallelGC
 android.useAndroidX=true
-kotlin.native.binary.memoryModel=experimental
+#kotlin.native.binary.memoryModel=experimental
```

```suggestion```

This can be removed, the "experimental" is the default since Kotlin 1.7.20

We can also remove the `kotlin.native.binary.freezing=disabled` in line 25.